src/docs/PML/PMLUserGuide.adoc

```diff
@@ -1564,6 +1564,130 @@ This example trains a Mnist model, starts a service instance where the trained m
+| Int (-1 means preview all the rows)
+|===
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
+
+You can now access the AutoFeat Page by clicking on the endpoint.
+
+You will be redirected to AutoFeat page which initially contains three tabs we describe in the following sections.
+
+=== Data Preview
+
+AutoFeat loads data from external sources. The dataset could be potentially very big. Initially, Only the 10 first data rows are displayed.
+
+The *Refresh* button enables users to see the last updates made on their data.
+[[_Data_preview]]
+image::AutoFeat_data_preview.png[align=center]
+
+=== Column summaries
+
+Whenever AutoFeat loads data from external sources, it also identifies the datatype of each column. AutoFeat does a great job at datatype recognition. Each decision can be overridden manually by the user, if required.
+
+AutoFeat also creates some summary statistics for each column. A table is displaying the missing values, minimum, maximum, mean and zeros for each numerical feature, and the cardinality (category counts) for each categorical feature.
+
+[[_Column_summaries]]
+image::AutoFeat_column_summaries.png[align=center]
+
+=== Edit column names and types
+A preview of the data is displayed in the *Edit Column Names and Types* as follows.
+
+[[_Edit_column_names_and_types]]
+image::AutoFeat_edit_column_names_and_types.png[align=center]
+
+It is possible to change a column information. These changes can include:
+
+- _Column Name_: There should rarely be a reason to change the field name.
+
+- _Column Type_: AutoFeat automatically recognizes the data type, so the default settings typically do not need to be changed.There are two different data types; *Categorical* and *Numerical*.
+
+- _Category Type_: Categorical variables can be divided into two categories; *Ordinal* such the categories have an inherent order and *Nominal* if the categories do not have any inherent order.
+
+- _Label_: Check this checkbox to select the label column. Label column is the feature about which we want to gain a deeper understanding.
+
+- _Coding Method_: The encoding method used for converting the categorical data values into numeric values. The value is set to *Auto* by default. Thereafter, the method best suited for encoding the categorical feature is automatically identified. The data scientist still has the power to override every decision and select another encoding method from the drop-down menu. Different methods are supported by AutoFeat such as *Label*, *OneHot*, *Dummy*, *Binary*, *Base N*, *Hash* and *Target*. Some of those methods require specifying additional encoding parameters. These parameters vary depending on the selected (e.g., the base and the number of components for BaseN and Hash, respectively, and the target column for Target encoding method). Some of those values are set by default, if no values are specified by the user.
```

Comment:
can he know which best method the algo has selected for him? when auto is enabled ?

Reply:
Not at this stage, we're planning to add it as a futur enhancement.